File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/import-and-initialize/java/import.java
import com.absmartly.sdk.*;

public class Example {
    static public void main(String[] args) {

        final ABsmartly sdk = ABsmartly.builder()
            .endpoint("https://your-company.absmartly.io/v1")
            .apiKey(System.getenv("ABSMARTLY_APIKEY"))
            .application("website")
            .environment("production")
            .build();
        // ...
    }
}

File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/import-and-initialize/elixir/import.ex
alias ABSmartly.SDK

{:ok, sdk} = SDK.new(
  endpoint: "https://your-company.absmartly.io/v1",
  api_key: System.get_env("ABSMARTLY_API_KEY"),
  application: "website",
  environment: "production"
)
